modules/common-fs2/src/main/scala/com/snowplowanalytics/snowplow/enrich/common/fs2/io/Clients.scala

```diff
@@ -58,7 +59,7 @@ object Clients {
       }
     }
 
-  def mkHttp[F[_]: ConcurrentEffect](
+  def mkHttp[F[_]: ConcurrentEffect: Timer]( //TODO it looks like it's used anywhere http client is needed in enrich, so same client everywhere. Maybe we should split clients and make retries only for iglu lookup...?
```

Comment:
I don't have a problem with letting all clients use this retry policy.  From what I understand, the error happens when a server closes a connection unexpectedly.  It seems that could affect any api call, not just iglu.

Reply:
Yeah I think we're OK.  It is used for:

- Telemetry.  Which uses the snowplow scala tracker and so retries requests already, even without this change.
- Metadata (experimental feature).  Same as telemetry, this uses the snowplow scala tracker to relay snowplow events.
- Remote adapter.  These should be idempotent GET requests.
- Downloading assets.  Which also does retries already, even before this change.